ProjFS.Linux/PrjFSLib.Linux.Managed/Interop/PrjFSLib.cs

```diff
@@ -12,9 +12,15 @@ internal static class PrjFSLib
         // TODO(Linux): revise library functions for Linux
         [DllImport(PrjFSLibPath, EntryPoint = "PrjFS_StartVirtualizationInstance")]
         public static extern Result StartVirtualizationInstance(
+            string storageRootFullPath,
             string virtualizationRootFullPath,
             Callbacks callbacks,
-            uint poolThreadCount);
+            uint poolThreadCount,
+            ref IntPtr mountHandlePtr);
```

Comment:
Nit: rename to `mountHandle`

Reply:
I used `mountHandlePtr` because this is a `**thing` into which the start function writes the `*thing`, whereas in the stop function it just takes the `*thing`.  So I'm inclined to stick with this unless it's highly non-idiomatic in C#; let me know what you think!

Reply:
It's definitely non-idiomatic, but I also think we can probably drop this :-). So let's discuss that point further.

Reply:
OK, done, thanks for the advice!